publish_to_hugging_face.py

```diff
@@ -0,0 +1,41 @@
+from app_utils import hugging_face_utils
+
+import argparse
+import logging
+import sys
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser(description="")
+  parser.add_argument("-e", "--experiment_dir", required=True, help="experiment dir", default=argparse.SUPPRESS)
+  parser.add_argument("-d", "--device", help="'cpu' or 'cuda:0', if the GPU device id is 0", default=argparse.SUPPRESS)
+  parser.add_argument("-a", "--api_key", help="Hugging Face API Key", default=argparse.SUPPRESS)
+  parser.add_argument("-r", "--repository_id", help="Hugging Face Repository ID", default=argparse.SUPPRESS)
+
+  parser_args, unknown = parser.parse_known_args(sys.argv)
+
+  #args = vars(parser.parse_args())
+
+  if "experiment_dir" in parser_args:
```

Comment:
`argsparser` will already raise an issue when calling `parse_known_args`, as `experiment_dir` is required.

Reply:
Done